examples/computer_vision/deformabledetr_coco_pytorch/model_def.py

```diff
@@ -71,9 +72,9 @@ def cross_slot_reduce(self, per_slot_metrics):
             coco_evaluator.eval_imgs[iou_type] = np.concatenate(
                 coco_evaluator.eval_imgs[iou_type], 2
             )
-            create_common_coco_eval(
-                coco_eval, coco_evaluator.img_ids, coco_evaluator.eval_imgs[iou_type]
-            )
+            coco_eval.evalImgs = list(coco_evaluator.eval_imgs[iou_type].flatten())
+            coco_eval.params.imgIds = list(coco_evaluator.img_ids)
+            coco_eval._paramsEval = copy.deepcopy(coco_eval.params)
```

Comment:
why is the copy necessary?  Seems like since this is example code, and what's going on seems opaque enough, maybe we should have some comments as to what is going on?